packages/vite/src/node/server/index.ts

```diff
@@ -923,6 +944,8 @@ export function resolveServerOptions(
     strict: server.fs?.strict ?? true,
     allow: allowDirs,
     deny,
+    cachedChecks:
+      server.fs?.cachedChecks ?? !!process.env.VITE_SERVER_FS_CACHED_CHECKS,
```

Comment:
Are we planning to keep this env var, or is it only for testing?

Reply:
I added it only for help with testing, for example if @sheremet-va would want to easily check the effect in Vitest. I think we could remove it later.